kernel/src/oops.rs

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! Kernel oops handling.
+//!
+//! In Asterinas, the Rust panic leads to a kernel oops. The kernel oops behave
```

Comment:
"a Rust panic leads to a kernel oops".

By the way, what is the plural form of oops?

Reply:
It's an informal exclamation word. I bet there's no plural form for it since it's not a noun.

Reply:
Better to quote it as "oops" I think.

Reply:
The plural form is "oopses", [according to the Linux documentation](https://docs.kernel.org/search.html?q=oopses).